doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,223 @@
+
+We are using ``stanford_hardi`` dataset. You can also use your own data.
+
+The workflow for the LPCA denoising requires the paths to the diffusion input file,
+as well as the b-values and b-vectors files.
+
+You may want to create an output directory to save the denoised data::
+
+    mkdir denoise_lpca_output
+
+Run the following command::
+
+    dipy_denoise_lpca data/stanford_hardi/HARDI150.nii.gz data/stanford_hardi/HARDI150.bval data/stanford_hardi/HARDI150.bvec --out_dir "denoise_lpca_output"
+
+This command will denoise the diffusion image and save it to the directory
+``denoise_lpca_output``.
+
+In case, you did not specify ``out_dir``, the denoised diffusion image will 
+be saved to a file named ``dwi_lpca.nii.gz`` by default, located in the 
+input directory also by default.
+
+.. |image1| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_original.png  
+   :scale: 100%
+   :align: middle
+.. |image2| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_denoise_LPCA.png
+   :scale: 100%
+   :align: middle
+
++--------------------+--------------------+
+|  Before Denoising  |  After Denoising   |
++====================+====================+
+|      |image1|      |      |image2|      |
++--------------------+--------------------+
+
+-----------------------------------
+Denoising using Marcenko-Pastur PCA
+-----------------------------------
+
+The PCA-based denoising algorithm exploits the redundancy across the diffusion-
+weighted images. This algorithm has been shown to provide an optimal compromise
+between noise suppression and loss of anatomical information for different 
+techniques such as DTI, spherical deconvolution and DKI.
+
+The basic idea behind the PCA-based denoising algorithms is to remove the components 
+of the data that are classified as noise. The Principal Components classification
+can be performed based on prior noise variance estimates or automatically based 
+on the Marcenko-Pastur distribution. In addition to noise suppression, the PCA
+algorithm can be used to get the standard deviation of the noise.
+
+We will use the ``sherbrooke_3shell`` dataset in DIPY to showcase this denoising method.
+As with any other workflow in DIPY, you can also use your own data!
+
+Create a directory where to save the denoised image (e.g.:
+``denoise_mppca_output``)::
+
+    mkdir denoise_mppca_output
+
+In order to run the MPPCA denoising method, we need to specify the location of the 
+diffusion data file, followed by the optional arguments. In this case, we will be 
+specifying the ``patch radius`` value and ``output directory``.
+
+So, we will run the command as::
+
+    dipy_denoise_mppca data/sherbrooke_3shell/HRADI193.nii.gz --patch_radius 10 --out_dir "denoise_mppca_output"
+
+This command will denoise the diffusion image and save it to the specified output directory.
+
+.. |image3| image:: https://github.com/dipy/dipy_data/blob/master/sherbrooke_3shell_original.png  
```

Comment:
you need to add `?raw=true` at the end of the URL.

Reply:
Addressed in 12ceeef